packages/neuron-wallet/src/services/tx/transaction-service.ts

```diff
@@ -427,12 +446,17 @@ export class TransactionsService {
           nftInfo = { type: NFTType.Receive, data: receiveNFTCell.typeArgs! }
         }
 
+        if (!txType) {
+          txType = value > BigInt(0) ? 'receive' : 'send'
```

Comment:
`txType` can be initialized at line 361 `let txType: string = ''`

Reply:
I means initialized like this: `txType = value > BigInt(0) ? 'receive' : 'send'`.Then you can delete this code.

Reply:
yes, that make sense